plugins/modules/cargo.py

```diff
@@ -58,15 +58,87 @@
     type: bool
     default: false
     version_added: 7.5.0
+  force:
+    description:
+      - Forcefully install package(s).
+      - Ignores many checks, and updates/installs forcefully.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.4.0
   state:
     description:
       - The state of the Rust package.
     required: false
     type: str
     default: present
     choices: [ "present", "absent", "latest" ]
+  features:
+    description:
+      - List of features to enable for the package.
+      - Do not use with multiple names.
+    type: list
+    elements: str
+    required: false
+    default: []
+    version_added: 8.4.0
+  all_features:
+    description:
+      - Enable all features for package(s).
+    type: bool
+    default: false
+    required: false
+    version_added: 8.4.0
+  directory:
+    description:
+      - Path to install the package(s) from.
+    type: path
+    required: false
+    version_added: 8.4.0
+  registry:
+    description:
+      - Registry to install the package(s) from.
+    type: str
+    required: false
+    version_added: 8.4.0
+  git:
+    description:
+      - Git URL to install the package(s) from.
+      - Can optionally either be used with C(branch), C(tag) or C(rev).
```

Comment:
Always use `O(...)` to refer to options.
```suggestion
      - Can optionally either be used with O(branch), O(tag) or O(rev).
```